SCClassLibrary/Common/Core/Object.sc

```diff
@@ -11,13 +11,9 @@ Object {
 		// to actually put things in the object you need to
 		// add them.
 	}
-	*newCopyArgs { arg ... args;
+	*newCopyArgs { | ... args, kwargs |
 		_BasicNewCopyArgsToInstVars
 		^this.primitiveFailed
-		// creates a new instance that can hold up to maxSize
```

Comment:
is all the info in this comment no longer applicable? if some of the comment is valid, perhaps updating/rewording it to fit the new functionality (or clarify it... offhand I'm not sure what it's referring to, or if it's mistakenly referring to slotted objects like `Array`?)

Reply:
It's a repeat of `new`, it's directly above it and this is a copy and paste. It just doesn't apply here, the size is fixed and the items are inserted.